example/fixtures.py

```diff
@@ -44,9 +45,19 @@ def exec_find(self, params):
 def create_django_bootstrap_container(
         request, name, wait_lines, command=None, single_container=False,
         publish_port=True, other_fixtures=()):
+    pods = request.getfixturevalue('pods')
+    # FIXME: there are probably better ways to skip these tests
+    if pods:
+        if request.fixturename in ['web_only_container', 'single_container']:
+            pytest.skip()
```

Comment:
Maybe we can tie into collection hooks or something?
https://docs.pytest.org/en/latest/writing_plugins.html#hook-function-validation-and-execution